compose/docker-compose.yaml

```
@@ -0,0 +1,7 @@
+version: "25.0.3"
+services:
+  app:
+    image: todo_app:prod
+    ports: 
+      - "8081:80"
```

This disagrees with the command in your Readme, and seems to be mismatched on ports again!
```suggestion
      - "8081:5000"
```

Remember we need the right hand side to match the port that the app is running on within the container, which for us is Flask's default at 5000